.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -96,47 +80,130 @@ runs:
         cd ../pytorch
         source /opt/intel/oneapi/compiler/latest/env/vars.sh
         rm -rf inductor_log
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 0 static 8 0 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 1 static 8 1 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 2 static 8 2 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 3 static 8 3 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 4 static 8 4 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 5 static 8 5 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 6 static 8 6 & \
-        bash inductor_xpu_test.sh ${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }} xpu 7 static 8 7 & wait
-        LOG_NAME=inductor_${{ inputs.suite }}_${{ inputs.dt }}_${{ inputs.mode }}_xpu_${{ inputs.scenario }}_all.log
-        for log in $(ls inductor_log/${{ inputs.suite }}/${{ inputs.dt }}/*card*.log); do cat $log >> inductor_log/${{ inputs.suite }}/${{ inputs.dt }}/${LOG_NAME}; done
-    - name: Test Results Overview (${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }})
+        # check param
```

Comment:
Suggest to split the param check as a standalone step, and write the valid value into $GITHUB_ENV and use them in below steps directly. 
```
echo "SUITES=huggingface,timms_model" >> ${GITHUB_ENV} 
echo "DTS=float32" >> ${GITHUB_ENV} 
...
```